main/setup-sentry-on-backend.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const Sentry = require('@sentry/node')
+const { BUILD_VERSION } = require('./consts')
+
+if (BUILD_VERSION.endsWith('-dev')) {
+  // Disable Sentry integration for dev builds
+} else {
```

Comment:
```suggestion
// Disable Sentry integration for dev builds
if (!BUILD_VERSION.endsWith('-dev')) {
```

It's not a logical branch, so I think just a comment is enough